cookbooks/new_ping.md

```diff
@@ -0,0 +1,98 @@
+Got some new data you want to send to us? How in the world do you send a new ping? Follow this guide
+to find out.
+
+Write Your Questions
+--------------------
+Do not try and implement new pings unless you know specifically what questions you're trying to
+answerer. General questions about "How do users use our product?" won't cut it - these need to be
+specific, concrete asks that can be translated to data points. This will also make it easier down
+the line as you start data review.
+
+Create a Schema
+---------------
+Use JSON Schema to start with. See the examples schemas in the
+[Mozilla Pipeline Schemas repo](https://github.com/mozilla-services/mozilla-pipeline-schemas/).
+This schema is just used to validate the incoming data; any ping that doesn't match the schema
+will be removed. Validate your JSON Schema using a
+[validation tool](https://jsonschemalint.com/#/version/draft-04/markup/json),
+
+We already have automatic deduping based on docId, which catches about 90% of duplicates and removes
+them from the dataset.
+
+Start a Data Review
+-------------------
+Data review for new pings is a more complicated than when adding new probes. See
+[Data Review for Focus-Event Ping](https://bugzilla.mozilla.org/show_bug.cgi?id=1347266)
+as an example. Consider where the data falls in the
+[Data Collection Categories](https://wiki.mozilla.org/Firefox/Data_Collection).
+
+Submit Schema to mozilla-services/mozilla-pipeline-schemas
+----------------------------------------------------------
+The first schema added should be the JSON Schema made in step 2.
+Make sure you add at least one example ping which the data can be validated against. Additionally,
+a [Parquet output](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/schemas/telemetry/core/core.9.parquetmr.txt)
+schema should be added. This would add a new dataset, available in [Re:dash](https://sql.telemetry.mozilla.org).
+
+Test your Parquet output in [Hindsight](https://pipeline-cep.prod.mozaws.net/) by testing
```

Comment:
Is testing in Hindsight still needed? The current mozilla-pipeline-schemas repo runs CI using the provided ping examples and schemas on every new PR

Reply:
Hmm, good point. Plus this would only work if the data is already being sent. I'll keep it but move it to the end of this section, because I do think it's an important reference.